### DISCUSSION TITLE
New logo

### DISCUSSION THREAD

**Discussion by katspaugh** [katspaugh/wavesurfer.js#2720] [General]
I've been playing around with Dalle-2, and came up with this logo that I kinda like:

<p align="center">
<img src="https://user-images.githubusercontent.com/381895/224699464-1a020c87-4a2d-4f02-901e-832c1ebbf09d.png" width="200" />
</p>

What do you folks think? cc @entonbiba

**Comment by entonbiba** [comment]
@katspaugh looks nice, I recreated and smoothed out the lines.
![wavesurferjs-logo](https://user-images.githubusercontent.com/5193884/225048871-953eeb4f-216e-41e7-8f2b-2fc6da5bbed4.png)

**Reply by sundayz** [reply]
nice, I like it. :)

**Reply by katspaugh** [reply]
Oh wow, thank you so much! ❤️

**Reply by katspaugh** [reply]
Adding it to the new website: https://github.com/katspaugh/wavesurfer.ts/pull/3
